mopidy/models/__init__.py

```diff
@@ -208,6 +208,8 @@ class Track(ValidatedImmutableObject):
     :type musicbrainz_id: string
     :param last_modified: Represents last modification time
     :type last_modified: integer or :class:`None` if unknown
+    :param live_stream: Represents if the URI must be streamed.
```

Comment:
Not sure if this should be named just `stream` or something else. One could also argue that it doesn't need to even be in the model. You could for instance have a `is_live(uri)` function in the playback provider similar to https://github.com/mopidy/mopidy/blob/develop/mopidy/backend.py#L199

Reply:
I implemented the solution with is_live, removing the live_stream from the model 
Yes it looks better,